cmd/snap/cmd_snap_op.go

```diff
@@ -382,9 +382,44 @@ func (x *cmdTry) Execute([]string) error {
 	return listSnaps([]string{name})
 }
 
+type cmdRevert struct {
+	Positional struct {
+		Snap string `positional-arg-name:"<snap>"`
+	} `positional-args:"yes"`
+}
+
+var shortRevertHelp = i18n.G("Revert the given snap to the previous state")
```

Comment:
We need to make a pass at some point moving the first word of the short help to a common form. The original suggestion was in the third person ("Reverts" in this case) but we have both already.